tests/config/pubsub/tests.yml

```diff
@@ -8,4 +8,6 @@ components:
       maxReadDuration: 1000
   - component: redis
     allOperations: true
+  - component: kafka
```

Comment:
similar config needs to be added to the workflow itself to run it ....

Reply:
I know. thanks.